### PR TITLE
⚡❓ AMP app integration

### PR DESCRIPTION
**Pull request by AileenCGN**
refs TryGhost/Ghost#7769

Adds Google AMP to `apps` in order to disable and enable it on request.

### CODE REVIEW

**Comment by acburdine** [comment]
@AileenCGN can you rebase this when you get a chance?

**Comment by AileenCGN** [comment]
@acburdine already on it, but it's going to be completely different... No checkbox in the settings, but an option in apps, just like Slack.

**Comment by acburdine** [comment]
Right, carry on then :)

**Comment by AileenCGN** [comment]
@kevinansfield this is ready for review!

We'll fix the styles a bit later, when we (@JohnONolan)  decided, how it should look like.

File: app/templates/settings/apps/amp.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            </div>
+            <div class="app-cell">
+                <h3>AMP</h3>
+                <p>Accelerated Mobile Pages</p>
+            </div>
+        </section>
+        <section class="app-subtitle">
+            <p>Enable <a href="https://ampproject.org">Google Accelerated Mobile Pages </a>for your site?</p>
+        </section>
+        {{model.amp}}
```
Comment (AileenCGN) [inline app/templates/settings/apps/amp.hbs]: 😬 Nope, thanks...

**Pull request merged**